configs/default/VGRC-VGOODRCF4.config

```diff
@@ -1,38 +1,69 @@
-# Betaflight / STM32F405 (S405) 4.1.0 Apr 30 2019 / 09:30:28 () MSP API: 1.42
+# Betaflight / VGOODRCF4 (VGF4) 4.0.2 May 30 2019 / 10:19:58 () MSP API: 1.41
 
 board_name VGOODRCF4
 manufacturer_id VGRC
 
 # resources
 resource BEEPER 1 C13
-resource MOTOR 1 A09
-resource MOTOR 2 A10
-resource MOTOR 3 B00
-resource MOTOR 4 B01
-resource MOTOR 5 C08
-resource MOTOR 6 C09
+resource MOTOR 1 A08
```

Comment:
Was this board ever sold to customers? If so, this will cause this board to stop working for all existing customers.

Reply:
Thanks for your remind. We don't sell  products with old version to the market, as when we test, we find it need to improve, so please replace with this new, thanks.